test/object-store/migrations.cpp

```diff
@@ -1203,7 +1316,7 @@ TEST_CASE("migration: Automatic") {
             // ObjectStore::delete_data_for_object(realm->read_group(), "DetailStudentStatus");
             Object old_obj(old_realm, "EmpDetails", 0);
             Object new_obj(new_realm, "EmpDetails", 0);
-
```

Comment:
the following whitespace changes seem unnecessary, is your editor doing this automatically?